src/cache/InvalidationPolicyCache.ts

```diff
@@ -41,9 +41,9 @@ export default class InvalidationPolicyCache extends InMemoryCache {
       policies: invalidationPolicies,
       entityTypeMap: this.entityTypeMap,
       cacheOperations: {
-        evict: this.evict.bind(this),
-        modify: this.modify.bind(this),
-        readField: this.readField.bind(this),
+        evict: (options) => this.evict(options),
```

Comment:
Can we spread here too in case the evict API is given more arguments?

Reply:
I originally did that, but it gives me a TS error!

![image](https://user-images.githubusercontent.com/616906/108090164-9d299080-707a-11eb-911f-538473cdec0c.png)

Reply:
Ah got it, yep looks like Apollo has removed the additional legacy args in a recent version. So we can remove them from the two type definitions here: https://github.com/NerdWalletOSS/apollo-invalidation-policies/blob/master/src/policies/types.ts#L83:L83